src/GitHub.Exports.Reactive/ViewModels/ViewModelItemContainer.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace GitHub.ViewModels
+{
+    /// <summary>
+    /// Simple container for complex objects that we stuff
+    /// into dropdowns and the likes
+    /// </summary>
+    public abstract class ViewModelItemContainer
+    {
+        public string Name;
```

Comment:
@shana If we're refactoring this, we might as well change this to be a read-only property right? I'm assuming that classes derived from would expect this value to be immutable judging from the lack of INPC? If so a ctor taking the name would be preferable.

Public fields are euurgh.